pulp_container/app/redirects.py

```diff
@@ -26,7 +27,9 @@ def redirect_to_content_app(self, content_type, content_id):
         Redirect to the content app.
         """
         return self.distribution.redirect_to_content_app(
-            f"{settings.CONTENT_ORIGIN}/pulp/container/{self.path}/{content_type}/{content_id}"
+            urljoin(
+                settings.CONTENT_ORIGIN, f"/pulp/container/{self.path}/{content_type}/{content_id}"
+            )
```

Comment:
The redirect here would fail in case `CONTENT_ORIGIN=None`, right?
What if we do something like
```
    host = settings.CONTENT_ORIGIN or f"{request.scheme}://{request.get_host()}"
    urljoin(
                host, f"/pulp/container/{self.path}/{content_type}/{content_id}"
            )
```

Reply:
The django documentation says it allows for relative urls (https://docs.djangoproject.com/en/4.2/topics/http/shortcuts/#redirect) so if `CONTENT_ORIGIN=None` this should return a relative path for the redirect. I assume that it works since I'm testing the setting being None in the S3 runner.

Reply:
ohh...nice!!
Thank you for the clarification!